newIDE/app/src/guidelines/index.js

```diff
@@ -298,25 +286,23 @@ export default class GuidelinePopOver extends PureComponent<Props, State> {
                         element: this.state.arrowRef,
                       },
                     }}
-                   
-                     //className={'guidelineArrowContainer'}
                     className="guidelineArrowContainer"
                   >
                     <div
                       ref={this._handleArrowRef}
                       className={'guidelineArrow'}
                     />
 
-                    <Paper elevation={24} style={styles.guidelineContainer}>
-                      <div style={styles.guidelineDescription}>
+                    <Paper elevation={24} className={"guidelineContainer"}>
+                      <div className={"guidelineDescription"}>
                         <Line>
                           <Typography variant="h5">
-                            {guidelines[this.state.index].title}
+                            <Trans>{guidelines[this.state.index].title}</Trans>
                           </Typography>
                         </Line>
                         <Line>
                           <Typography wrap="true">
-                            {guidelines[this.state.index].description}
+                            <Trans>{guidelines[this.state.index].description}</Trans>
```

Comment:
This won't work. `<Trans>` is uniquely for *extracting* english sentences to be translated and shown on screen.
Here, it's properly shown on screen BUT it's just a variable. If your variable is just a number or a name entered by the user it's fine. But here, it's an entire english sentence! 
It won't be extracted and not translated as considered as a variable.

Instead, you must:
- mark description with `t`: 

```
{
  ...
  description: t`Bla bla`
}
```

Show it with i18n._:

```
<i18n>
{({ i18n }) => (
  <Something>
    <SomethingElse>
    {i18n._(description)}
    </SomethingElse>
  </Something>
)}
<i18n>
```